src/test/java/io/ticktok/TicktokTest.java

```diff
@@ -18,22 +17,23 @@
 public class TicktokTest {
 
     private static final String EVERY_5_SECONDS = "every.5.seconds";
-    private TicktockServiceStub ticktockServiceStub;
+    private static TicktockServiceStub ticktockServiceStub;
 
-    @Before
-    public void init() throws IOException {
+    @BeforeAll
+    public static void init() {
         ticktockServiceStub = new TicktockServiceStub(9999, true);
     }
 
     @Test
-    public void registerNewClock() throws IOException {
+    public void registerNewClock() {
         // TODO - TicktokOptions should be builder
```

Comment:
don't leave todos if you need to add a chore to pivotal

Reply:
fixed